apps/docs/src/app/core/component-docs/object-marker/examples/object-marker-text-example.component.html

```diff
@@ -1,9 +1,9 @@
-<span fd-object-marker aria-label="Object Marker with Text" label="Draft"></span>
-<span fd-object-marker aria-label="Object Marker with Text" label="Request"></span>
+<span fd-object-marker aria-label="Draft Text" label="Draft"></span>
+<span fd-object-marker aria-label="Request Text" label="Request"></span>
 <a
```

Comment:
I believe you should have href o.w. screen readers might complain?

Reply:
added.